application/routes/term.py

```
@@ -35,30 +37,34 @@
 def start_terminal():
     session_id = request.json.get('sessionID')
     load_check = request.json.get('loadCheck', True)
-
+    logger.debug(f"Term: Starting terminal session: {session_id}")
```

Ruff

Logging statement uses f-string

(G004)